cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1188,6 +1198,10 @@ class SlurmSettingsSchema(BaseSchema):
 
     scaledown_idletime = fields.Int(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
     dns = fields.Nested(DnsSchema, metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
+    queue_update_strategy = fields.Str(
+        validate=validate.OneOf([strategy.value for strategy in QueueUpdateStrategy]),
+        metadata={"update_policy": UpdatePolicy.IGNORED},
```

Comment:
because we don't want to trigger an update if only the param QueueUpdateStrategy is change. QueueUpdateStrategy value isn't backed by any resource of the cluster